src/javascripts/components/hub/hub.js

```diff
@@ -0,0 +1,19 @@
+import airportComponent from './airportComponent';
+import airportData from '../../helpers/data/hubData';
+import utils from '../../helpers/utils';
+
+const printAirports = () => {
+  airportData.getAllAirports()
+    .then((airports) => {
+      let domString = '';
+      domString += '<h1>Airports</h1>';
```

Comment:
you can remove this - its going in the navbar in another ticket

Reply:
done